module2.js

```diff
@@ -111,7 +135,9 @@ function question6(promise) {
 
 function question7() {
   return fetch('https://example.com')
-  // TODO
+    .then((response) => {
+      return response.text();
```

Comment:
A similar problem in the test setup here - it was intended that you call the `.json()` method to get the parsed result, then return the `hello` prop from the resulting object.